scapy/arch/windows/__init__.py

```diff
@@ -772,6 +772,16 @@ def get_working_if():
         # no route
         return scapy.consts.LOOPBACK_INTERFACE
 
+conf.iface = get_working_if()
```

Comment:
Sorry i was unclear.
Every thing is good except `conf.iface = get_working_if()`

Reply:
Do you want `conf.iface` to be set at runtime (even if you don't need it)? Where would you want this call to be made instead?

Reply:
It's already done:
https://github.com/secdev/scapy/blob/master/scapy/route.py#L195

Reply:
OK so it's safe to remove it?